src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -10,5 +10,6 @@
     public static final String MESSAGE_INVALID_FLASHCARD_DISPLAYED_INDEX = "The flashcard index provided is invalid";
     public static final String MESSAGE_FLASHCARD_LISTED_OVERVIEW = "%1$d  flash cards listed!";
     public static final String MESSAGE_UNKNOWN_TEST_COMMAND = "Unknown test command";
-
+    public static final String MESSAGE_EXPORT_IO_EXCEPTION = "There was an error in writing to the file.\n"
+            + "Please ensure your file path doesn't contain any illegal characters.";
```

Comment:
You can combine the two strings to something like "A\nB" instead of "A\n" + "B"

Reply:
But ignore this if the line is too long (120char?)

Reply:
Yep, the line is too long if I don't break it.